sormas-backend/src/main/java/de/symeda/sormas/backend/travelentry/TravelEntryFacadeEjb.java

```diff
@@ -351,8 +276,14 @@ public TravelEntryDto toDto(TravelEntry entity) {
 		return dto;
 	}
 
-	private TravelEntry fillOrBuildEntity(@NotNull TravelEntryDto source, TravelEntry target) {
-		target = DtoHelper.fillOrBuildEntity(source, target, TravelEntry::new, true);
+	@Override
+	public TravelEntryReferenceDto toRefDto(TravelEntry travelEntry) {
```

Comment:
Check the other `toRefDto` methods, they call the static `toReferenceDto` method here. To be consistent, please implement it here.

Reply:
again, i totally agree and should be done only that if i remove the static methods that propagates and i need to remove static everywhere (in all ejb's) - which i am all for it only not with this ticket